css/style.css

```diff
@@ -0,0 +1,75 @@
+html {
+  font-size: 100%;
+  overflow-y: scroll;
+}
+
+body {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 16px;
+  line-height: 1.61;
+  font-weight: 400;
+  width: 1024px;
+  font-family: 'Fantasy', fantasy;
```

Comment:
it's recommended to write web safe fonts and font type after non-standard font (for ex. font-family: "Helvetica", "Arial", sans-serif;)
http://codeguide.academy/html-css.html#css-font-variant